docs/en/connector/source/Fake.mdx

```diff
@@ -116,6 +116,39 @@ source {
     FakeSourceStream {
         result_table_name = "fake"
         field_name = "name,age"
+        //mock_data_enable = true
```

Comment:
Please write correct doc.

Reply:
I‘m sorry to push the pom.xml. I will resove it.

Reply:
The docs, let me learn some alout mdx format.